CONTRIBUTING.md

```diff
@@ -288,6 +288,28 @@ node seed
 npm run develop
 ```
 
+### Setup freeCodeCamp via Docker
+
+Create docker-compose.yml from sample file
+```bash
+cp sample.docker-compose.yml docker-compose.yml
+```
+Open .env and rename host from `localhost` to `mongo`
```

Comment:
Add explicit instructions on what is to be changed

`change MONGOHQ_URL='mongodb://localhost:27017/freecodecamp' to MONGOHQ_URL='mongodb://mongo:27017/freecodecamp'`

Reply:
would this not be better handled in the `compose` file under `environment`?

Reply:
Yup, that should be OK.